src/Microsoft.AspNetCore.Authentication/RemoteAuthenticationHandler.cs

```diff
@@ -89,9 +89,32 @@ public abstract class RemoteAuthenticationHandler<TOptions> : AuthenticationHand
 
         protected abstract Task<AuthenticateResult> HandleRemoteAuthenticateAsync();
 
-        protected override Task<AuthenticateResult> HandleAuthenticateAsync()
+        protected override async Task<AuthenticateResult> HandleAuthenticateAsync()
         {
-            return Task.FromResult(AuthenticateResult.Fail("Remote authentication does not support authenticate"));
+            // Most RemoteAuthenticationHandlers will have a PriorHandler, but it might not be set up during unit tests.
+            if (PriorHandler != null)
+            {
+                var authenticateContext = new AuthenticateContext(Options.SignInScheme);
+                await PriorHandler.AuthenticateAsync(authenticateContext);
+                if (authenticateContext.Accepted)
```

Comment:
Well given that previously asking for Facebook would never give you any identity anyways, yes... now remote schemes will give you the 'sign in scheme' identity when you ask.  If you are sharing sign in schemes, that may give you someone else's identity...  Kind of weird but I think it makes sense.